### ISSUE TITLE
Feedback from Sarah Watts and Miliban Keyim 

### ISSUE THREAD

**Issue by hfboyce** [UBC-MDS/DSCI_522-Age-and-Unemployment-Rates#53]
[Sarah Watts](https://github.com/smwatts) @smwatts
[Miliban Keyim](https://github.com/mkeyim) @mkeyim

The following was feedback and recommended changes from the students listed above in Lab 3.  

- Knit and Markdown are not dependencies and should be removed from the Readme file. 
- Instead of mentioning "libraries" actually, link the GitHub repository as a hyperlink.   

- Change the wording for the objective, this was found confusing and hard to follow. The recommendation that was provided was : 
 “For a set of developed countries, we will be determining if the mean unemployment rate changes based on age group”

- Move the assumptions above the analysis since the major issue with the data is that we are assuming that each gender is equally important in the analysis. 

- Suggested to pick between histograms or the violin jitter plot as the two are redundant. 

- Take out a sentence in "Assumptions" as it is repeating already mentioned residual assumptions.

**Comment by hfboyce** [comment]
I will be tackling these one by one tonight

**Comment by hfboyce** [comment]
fixed most of these in [#54](https://github.com/UBC-MDS/DSCI_522-Age-and-Unemployment-Rates/pull/54)

**Comment by cheukman1207** [comment]
I agree with all of the changes in that pull request and nice choice on picking the violin plot. One picky thing is that the word "thirdly" in one of the changes should've been a "Thirdly" with a captal T, which should be an easy fix. Nice work!
